src/flowTypes.js

```diff
@@ -338,3 +338,10 @@ export type MultiputPart = {
 export type MultiputData = {
     part?: MultiputPart
 };
+
+export type AccessStats = {
+    preview_count: number,
```

Comment:
any reason why these are _ based?

Reply:
Or will be writing V2 API that will return these?

Reply:
Yeah, this API will need to be built. Following the styling of other API's they tend to use _ instead of camelCase